support-frontend/assets/components/directDebit/directDebitActions.ts

```diff
@@ -116,76 +119,76 @@ const setDirectDebitFormPhase = (phase: Phase): Action => ({
 	phase,
 });
 
-function payDirectDebitClicked(): (...args: any[]) => any {
-	return (
-		dispatch: (...args: any[]) => any,
-		getState: (...args: any[]) => any,
-	) => {
+const directDebitErrorMessages = {
+	invalidInput:
+		'Your bank details are invalid. Please check them and try again',
+	incorrectInput:
+		'Your bank details are not correct. Please check them and try again',
+	default: 'Oops, something went wrong, please try again later',
+};
+
+type DirectDebitConfirmationResponse = {
+	accountValid: boolean;
+};
+
+function payDirectDebitClicked(): (
+	dispatch: Dispatch<Action>,
+	getState: () => CheckoutState,
+) => Promise<Action> {
```

Comment:
I was wondering if these thunks need to return anything? I feel like I usually see a thunks with the type signature

```ts
(dispatch: Dispatch, getState: () => CheckoutState) => void
```